tests/bud.bats

```diff
@@ -1981,6 +1981,32 @@ _EOF
   test -s $root/etc/passwd
 }
 
+@test "build using --layer-label and test labels on intermediate images" {
+  # Remove all images so no intermediate images are present
+  run_buildah rmi --all -f
+  _prefetch alpine
+  run_buildah --version
+  local -a output_fields=($output)
+  buildah_version=${output_fields[2]}
+  label="l_$(random_string)"
+  labelvalue="v_$(random_string)"
+
+  run_buildah build --no-cache --layers --layer-label $label=$labelvalue --layer-label emptylabel $WITH_POLICY_JSON -t exp -f $BUDFILES/simple-multi-step/Containerfile
+
+  # Final image must not contain this label
+  run_buildah inspect --format '{{ index .Docker.Config.Labels "'$label'"}}' exp
+  expect_output "" "label on actual image"
+
+  # All intermediate images must have this label
+  want_output='map["emptylabel":"" "io.buildah.version":"'$buildah_version'" "'$label'":"'$labelvalue'"]'
+  run_buildah images -a --format '{{.ID}}' --filter intermediate=true
+  for image in "${lines[@]}";do
+	run_buildah inspect --format '{{printf "%q" .Docker.Config.Labels}}' $image
+	expect_output "$want_output" "expected .Docker.Config.Labels to match with '$want_output'"
```

Comment:
I'm sorry, I don't think I was clear. My request was to include `$image` in the test-failure message, so that someone examining a test failure could easily see what happened. This change makes everything worse: you don't need to duplicate `$want_output`, because that will already be shown in the failure message.

A really good habit to get into is to inject failures into your tests. Like, change to `expect_output "sdfsdf$want_output"` and look at the failure message. If this test were to fail, would that error message give you what you need to debug it? How could you make it better?